src/accessories/garageDoorOpener.ts

```diff
@@ -238,15 +241,16 @@ export const setupGarageDoorOpener = (
               assignCurrentDoorState(CurrentDoorState.OPEN);
               if (autoCloseDelay) {
                 await waitFor(`${deviceId}.pending`, autoCloseDelay);
-                assignCurrentDoorState(CurrentDoorState.CLOSED);
+                const targetDoorState = service.getCharacteristic(Characteristic.TargetDoorState);
```

Comment:
This would trigger a new `emit` and re run in `.on(CharacteristicEventTypes.SET`